src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import ReactDOM from 'react-dom';
-import './Styles/index.css';
+import './Styles/base.scss';
```

Comment:
Bringing in the reset file is key NOW as we move forward, so we don't have to retroactively 'fix' css things.